src/app/api/chapter/[chapterId]/route.ts

```diff
@@ -31,5 +42,40 @@ export const DELETE = withSessionAndRole(["ADMIN"], async ({ params }) => {
     )
   );
 
-  return NextResponse.json("ok");
+  await prisma.chapterRequest.delete({
+    where: {
+      id: chapter.chapterRequestId,
+    },
+  });
+
+  await prisma.userRequest.deleteMany({
+    where: {
+      chapterId: chapterId,
+    },
+  });
```

Comment:
1. Add a relation of userRequest to chapter
2. Add onDelete cascade so that this call isn't necessary

Reply:
By doing this we can remove this deleteMany call